code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -391,11 +382,6 @@ var/global/list/robot_modules = list(
 	modules += new /obj/item/extinguisher(src) // For navigating space and/or low grav, and just being useful.
 	modules += new /obj/item/device/flash(src) // Non-lethal tool that prevents any 'borg from going lethal on Crew so long as it's an option according to laws.
 	modules += new /obj/item/crowbar/robotic(src) // Base crowbar that all 'borgs should have access to.
-	modules += new /obj/item/pen/robopen(src)
```

Comment:
Pen is important for the Construction module to be able to name doors, perhaps?

Reply:
Good point, re-added pen to both modules.

Reply:
Should maint drones be given the pen as well?